scripts/testpop/testpop.go

```diff
@@ -61,9 +66,10 @@ func main() {
 		ringpop.Channel(ch),
 		ringpop.Identity(*hostport),
 		ringpop.Logger(bark.NewLoggerFromLogrus(logger)),
-		ringpop.SuspectPeriod(5*time.Second),
-		ringpop.FaultyPeriod(5*time.Second),
-		ringpop.TombstonePeriod(5*time.Second),
+
+		ringpop.SuspectPeriod(time.Duration(*suspectPeriod)*time.Second),
```

Comment:
you can't multiply an int by a duration.
when you hav 5*time.Second, the literal 5 is automatically converted by the compiler